src/__screenshot_tests__/fixed-footer-screenshot-test.tsx

```diff
@@ -37,3 +37,35 @@ test('Fixed footer with static position when height is smaller than the footer i
     const imageOverflowScrolled = await page.screenshot({fullPage: false});
     expect(imageOverflowScrolled).toMatchImageSnapshot();
 });
+
+test('Fixed footer with safe inset at the bottom', async () => {
+    const page = await openStoryPage({
+        id: 'layout-fixed-footer-layout--more-complex-footer',
+        device: 'MOBILE_IOS_INSET',
+    });
+
+    const image = await page.screenshot();
+    expect(image).toMatchImageSnapshot();
+
+    await (await screen.findByRole('button', {name: 'Load more text'})).click();
```

Comment:
maybe you could use `.click({clickCount: 3})` instead of repeating the same line 3 times